rapids-cmake/cpm/package_override.cmake

```diff
@@ -85,7 +85,15 @@ function(rapids_cpm_package_override filepath)
     include(FetchContent)
     include("${rapids-cmake-dir}/cpm/detail/package_details.cmake")
     rapids_cpm_package_details(${package_name} version repository tag shallow exclude)
-    FetchContent_Declare(${package_name} GIT_REPOSITORY ${repository} GIT_TAG ${tag})
+
+    include("${rapids-cmake-dir}/cpm/detail/generate_patch_command.cmake")
+    rapids_cpm_generate_patch_command(${package_name} ${version} patch_command)
+
+    FetchContent_Declare(${package_name}
+                         GIT_REPOSITORY ${repository}
+                         GIT_TAG ${tag}
+                         GIT_SHALLOW ${shallow}
+                         PATCH_COMMAND ${patch_command} EXCLUDE_FROM_ALL ${exclude})
```

Comment:
There's no `rapids_cpm_display_patch_status` here. Is that intentional?

Reply:
Yes it is intentional

When you call `FetchContent_Declare` the entries are stored to be executed in the future by `FetchContent_Populate` via something like `cpmAdd/FindPackage`. We pre-populate the entries first to override any that might exist in the `CPM` call.

We can't inject a status call since that has to occur after `FetchContent_Populate` executes since it reads a file that our patch command writes out.

Reply:
Great, thank you for that explanation.